src/api/app/models/workflow/step.rb

```diff
@@ -64,14 +66,19 @@ def source_project_name
     step_instructions[:source_project]
   end
 
-  def target_package_name
+  def target_package_name(short_commit_sha: true)
```

Comment:
With the default being a short commit SHA, it could potentially break the `branch_package` and `link_package` steps if there are commits with the SHA in different SCM repositories. Two packages in a project could then have the same name, this would raise an exception.

Reply:
Yeah, I noticed this before. That's why I marked at draft again. I have to push more changes. :+1: